faq/serverless-jobs.mdx

```diff
@@ -17,6 +17,69 @@ Scaleway Serverless Jobs is a fully managed service that enables efficient execu
 
 <Concept>
 
+## How am I billed for Serverless Jobs?
+
+### Principles
+
+Serverless Jobs are billed on a pay-as-you-go basis, strictly on resource consumption (Memory and CPU). 
+
+* **Memory consumption:** The memory consumption is obtained by multiplying the memory tier chosen by the job run duration.
+* **vCPU consumption:** The vCPU consumption is obtained by multiplying the vCPU tier chosen by the job run duration.
+
+### Prices 
+
+* **Memory consumption:** €0.10 per 100 000 GB/s, and we provide a **400 000 GB/s free tier** per account and per month.
+
+    | Memory   | Price per second  |
+    |--------- |------------------ |
+    | 128 MB   | €0.000000125      |
+    | 256 MB   | €0.00000025       |
+    | 512 MB   | €0.0000005        |
+    | 1024 MB  | €0.000001         |
+    | 2048 MB  | €0.000002         |
+    | 3072 MB  | €0.000003         |
+    | 4096 MB  | €0.000004         |
+
+* **vCPU consumption:** €1.00 per 100 000 vCPU/s, and we provide a **200 000 vCPU/s free tier** per account per month.
+
+    | CPU         | Price per second  |
+    |------------ |------------------ |
+    | 0.07 vCPU   | €0.0000007        |
+    | 0.14 vCPU   | €0.0000014        |
+    | 0.28 vCPU   | €0.0000028        |
+    | 0.56 vCPU   | €0.0000056        |
+    | 1.12 vCPU   | €0.0000112        |
+
+### Example
+
+| Criteria                   | Value            |
+|--------------------------- |------------------|
+| Monthly duration           | 432 000 s (120h) |
+| Amount of memory allocated | 2048 MB          |
+| Amount of vCPU allocated   | 1120 mVCPU       |
+| Free tier                  | Yes              |
+
+#### Price calculation
+
+* **Memory consumption**
+    * *Allocated Memory conversion:* 2048 MB = 2 GB
+    * *Resources consumption:* 432 000 s * 2 GB = 864 000 GB/s
+    * *Free tier:* 400 000 GB/s
+    * *Billed resources:* 864 000 - 400 000 = 464 000 GB/s
+    * *Cost:* 464 000 * €0.0000010 = **€0,464**
+* **vCPU consumption**
+    * *Allocated vCPU conversion:* 1120mVCPU = 1.12 vCPU
+    * *Resources consumption:* 432 000 s * 1.12 vCPU =  483 840 vCPU/s
+    * *Free tier:* 200 000 vCPU/s
+    * *Billed resources:* 483 840 - 200 000 = 283 840 vCPU/s
+    * *Cost:* 283 840 * €0.0000100 = **€2,8384**
```

Comment:
```suggestion
    * *Cost:* 283 840 * €0.0000100 = **€2.8384**
```

Reply:
I wonder if displaying the price rounded to the closest cent would be clearer? Maybe in brackets?